action.yaml

```diff
@@ -284,14 +284,14 @@ runs:
 
             blackduck_message=":arrow_right: Result  :${{ steps.all_icons.outputs.blackduck_scan }}:"
 
-        elif [ "${{ github.event_name }}" == "pull_request" ]; then
+        elif [ "${{ github.event_name }}" == "pull_request" ] || ["${{ github.event_name }}" == "pull_request_target" ]; then
```

Comment:
Have you tested this with all event types (Pull Request, Pull Request Target, Manual Trigger, Push)? Whether it is working fine or not, **please paste the links to the test cases here.**